content/en/data_streams/schema_tracking.md

```diff
@@ -8,7 +8,7 @@ title: Schema Tracking
 </div>
 {{% /site-region %}}
 
-<div class="alert alert-info">Schema tracking is in <b>beta</b> for Java services using Protobuf and Avro.</div>
+<div class="alert alert-info">Schema tracking is in <b>beta</b> for Java services using Protobuf and Avro. If you're interested in other languages and schemas, let us know here <a href="https://www.datadoghq.com/private-beta/schema-tracking/">here</a>. </div>
```

Comment:
```suggestion
<div class="alert alert-info">Schema tracking is in <b>beta</b> for Java services using Protobuf and Avro. If you're interested in other languages and schemas, complete the form <a href="https://www.datadoghq.com/private-beta/schema-tracking/">here</a>. </div>
```

Reply:
thanks for the suggestion! I originally opted for "let us know here" since "complete this form" might seem like a lot of work for customers. 

is there any other language we could use?

Reply:
Thanks for confirming. Would "reach out here" potentially work?

Reply:
Reach out here sounds great!